### PR TITLE
Fix a crash when objects are used after being deleted

### PR DESCRIPTION
**Pull request by D8H**
Fixes https://github.com/4ian/GDevelop/issues/5861

Tested with this event:
image

### CODE REVIEW

File: GDJS/Runtime/CustomRuntimeObject.ts [hunk 1]
```diff
@@ -88,11 +88,11 @@ namespace gdjs {
       }
     }
 
-    onDestroyFromScene(parent: gdjs.RuntimeInstanceContainer): void {
+    onDeletedFromScene(parent: gdjs.RuntimeInstanceContainer): void {
       // Let subclasses do something before the object is destroyed.
       this.onDestroy(parent);
```
Comment (4ian) [inline GDJS/Runtime/CustomRuntimeObject.ts:93]: Is this call valid/well named?

**Review by 4ian — APPROVED** [review]
Just a doubt about a call to onDestroy inside custom objects

**Comment by D8H** [comment]
> Just a doubt about a call to onDestroy inside custom objects

It's the EBO life-cycle function (the same as for event-based behaviors).

**Comment by fizzingpop** [comment]
Can confirm this fixed my issue. Good work!

**Pull request merged**